shims/common/src/main/java/org/apache/hadoop/hive/thrift/HadoopThriftAuthBridge.java

```diff
@@ -391,6 +396,32 @@ public TTransportFactory createTransportFactory(Map<String, String> saslProps)
     }
 
     /**
+     * Create a TTransportFactory that, upon connection of a client socket,
+     * negotiates a Kerberized SASL transport with PLAIN mechanism.
+     * The resulting TTransportFactory can be passed as both the input and output
+     * transport factory when instantiating a TThreadPoolServer, for example.
+     *
+     * @param saslProps Map of SASL properties
+     */
+    public TTransportFactory createPlainTransportFactory(Map<String, String> saslProps)
```

Comment:
You are right. In this function, saslProps is not used.
However, I keep this parameter for the consistency of the code.
HadoopThriftAuthBridge.Server class has two methods (createTransportFactory() and createPlainTransportFactor()) to create Kerberos SASL transport.